### PR TITLE
Auto OCP-35341

### PR DESCRIPTION
**Pull request by huiran0826**
Logs: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2493/console

@zhaozhanqi @anuragthehatter @weliang1 @rbbratta

### CODE REVIEW

File: features/networking/egress-ingress.feature [hunk 1]
```diff
@@ -464,3 +464,24 @@ Feature: Egress-ingress related networking scenarios
...
+  Scenario: EgressNetworkPolicy maxItems is 1000
+    Given I have a project
+    Given I obtain test data file "networking/egressnetworkpolicy/egressnetworkpolicy_1000.yaml"
+    When I run the :create admin command with:
+      | f | egressnetworkpolicy_1000.yaml |
+      | n | <%= project.name %>           |
+    Then the step should succeed
+    And the output should contain:
+      | egressnetworkpolicy.network.openshift.io/default created |
+    And admin ensures "default" egress_network_policy is deleted
```
Comment (weliang1) [inline features/networking/egress-ingress.feature:480]: @huiran0826 Not sure when creating the EgressNetworkPolicy the new iptable rule will also be created. If this is the case, besides checking EgressNetworkPolicy maxItems is 1000, should we also check 1000 new iptable rules are also created?

Comment (liangxia) [reply features/networking/egress-ingress.feature:480]: Any updates/follow up ?

Comment (huiran0826) [reply features/networking/egress-ingress.feature:480]: I did a check and there is no iptable rule created when creating the EgressNetworkPolicy. This case is from bug https://bugzilla.redhat.com/show_bug.cgi?id=1879910. And the fix is just bump the maxItems from 50 to 1000. Thanks for checking!

**Pull request merged**